R/doAzureParallel.R

```diff
@@ -36,7 +36,11 @@ registerDoAzureParallel <- function(cluster) {
 
 workers <- function(data) {
   id <- data$poolId
-  pool <- rAzureBatch::getPool(id)
+
+  config <- getConfiguration()
```

Comment:
rAzureBatch::getConfiguration?

Reply:
getConfiguration is in the doAzureParallel package, not in rAzureBatch